simulate/parameters.py

```diff
@@ -84,20 +84,29 @@ def build_population(agent_type, parameters, seed=None, dphm=1500):
 
 ### TODO: Population generators that take parameters like CRRA, DisCFac
 
+init_portfolio["cycles"] = 0  # NEED THIS FOR INFINITE HORIZON
+init_portfolio["PermGroFac"] = [1.0]  # no drift in perm income
+# risk free return, set to 1 to focus on equity premium
+init_portfolio["Rfree"] = 1.0
+init_portfolio["RiskyAvg"] = 1.05  # eq_prem is RiskyAvg - Rfree = 0.05
+init_portfolio["LivPrb"] = [1.0]  # no death
+
 lucas0_agent_population_params = init_portfolio.copy()
 
-lucas0_agent_population_params.update({
-    "cycles": 0,  # issue 186
-    "aNrmInitStd": 0.28901524, # calculated using dashboard from default init_portfolio stats
-    #"aNrmInitMean": 6
-    # "LivPrb": [0.98**0.25], -- Should this be adjusted?
-    "PermGroFac": 1.0,
-    "Rfree": 1.0,
-    ### These are placeholders that will be set when the system is set up.
-    "CRRA": 6,
-    "DiscFac": 0.90,
-    "ex_post": None,  # ex post heterogeneous parameters over which to merge solutions
-})
+lucas0_agent_population_params.update(
+    {
+        "cycles": 0,  # issue 186
+        "aNrmInitStd": 0.28901524,  # calculated using dashboard from default init_portfolio stats
+        # "aNrmInitMean": 6
+        # "LivPrb": [0.98**0.25], -- Should this be adjusted?
+        "PermGroFac": 1.0,
+        "Rfree": 1.0,
+        ### These are placeholders that will be set when the system is set up.
+        "CRRA": 5,
```

Comment:
This value is 6 in the Numerical notebook. Which do you mean?

Reply:
it is 5 in the default so let's go with 5